test/workers/repository/process/lookup/index.spec.js

```diff
@@ -52,7 +52,7 @@ describe('manager/npm/lookup', () => {
         .reply(200, qJson);
       expect((await lookup.lookupUpdates(config)).updates).toMatchSnapshot();
     });
-    it('returns only one update if grouping', async () => {
+    it('returns two updates if grouping but separateMajorMinor=true', async () => {
```

Comment:
Is there a test for when there are two updates and `separateMajorMinor=false`?

Reply:
@hbetts added now